src/intelligence_layer/connectors/studio/studio.py

```diff
@@ -106,6 +107,39 @@ class GetBenchmarkResponse(BaseModel):
     updated_by: str | None
 
 
+class PostBenchmarkExecutionRequest(BaseModel):
```

Comment:
How about just PostBenchmarkExecution for clairty? This one sound like we are asking Studio to do the execution for us

Reply:
yes good idea